doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -2602,169 +2604,239 @@ For more information about the PSA Crypto implementations in the |NCS|, see :ref
 
       .. tab:: nRF52 Series
 
```

Comment:
This table has inconsistent coloring. cc3xx has a grey topbar, while Oberon has a white one. 
<img width="927" height="596" alt="image" src="https://github.com/user-attachments/assets/7fc5a2e2-e4ed-45d6-874a-09bada22d811" />